halloffame.php

```diff
@@ -28,18 +22,43 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" type="text/css" href="halloffame.css.css">
+    <link rel="stylesheet" type="text/css" href="halloffame.css">
+
     <title>halloffame</title>
 </head>
 <body>
-<h1> FOOTBALLERS HALL OF FAME  </h1>
-<div id="container"
+<div class="navTop">
+    <div class="<img src="halloffame.png"></div>
+    <div class="title">
+        <h1> FOOTBALLERS HALL OF FAME </h1>
+    </div>
+</div>
 
+<header>
+<class="logo"> <img src="football.jpg" alt= "hall of fame logo"/>
+</header>
+
```

Comment:
Could add another container here, maybe <main></main> to wrap around this last section? Just an idea to keep it consistent with the rest of your code that is wrapped in <header></header>?
Would keep the two sections separate.